examples/fuse_examples/imaging/classification/cmmd/conf/config.yaml

```diff
@@ -5,6 +5,8 @@ paths:
   cache_dir : ${hydra:runtime.cwd}/examples/CMMD_cache_dir
```

Comment:
just a tip, because we spoke about it.
If you want to use an environment variable, this is the way:
`some_var: ${oc.env:YOUR_ENV_VAR_NAME}`

And if you want to do something custom in python, like if you have a function `get_path(bla1, bla2)` that returns some custom path, then you can write above the @hydra decorator:
`OmegaConf.register_new_resolver("your_custom_resolver_name", get_path(bla1,bla2))` 
and then in the config file:
`some_var: ${your_custom_resolver_name:}`